lib/simulator-xcode-6.js

```diff
@@ -290,6 +291,10 @@ class SimulatorXcode6 extends EventEmitter {
     if (!this.connectHardwareKeyboard) {
       args.push('-ConnectHardwareKeyboard', '0');
     }
+
+    // Do the 'Touch ID Enroll' key bindings before the Simulator starts
+    await setTouchEnrollKey();
```

Comment:
EDIT: On further reading, I can see that there actually would be test cases where you need to test cases where touchID is enrolled and cases where touchID is not unenrolled. So it does make sense to have both.